spec/controllers/idv/review_controller_spec.rb

```diff
@@ -0,0 +1,148 @@
+require 'rails_helper'
+
+describe Idv::ReviewController do
+  render_views
+
+  let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
```

Comment:
I have tried just `User.new` but because there is an access to `user_session` within the `create` method, Devise seems to want a real db user both signed in and present in the DB.